src/v2/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -110,9 +108,7 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
     phoneNumberError: "",
     phoneNumberTouched: false,
     selectedSavedAddress: defaultShippingAddressIndex(this.props.me),
-    editAddressIndex: -1,
     saveAddress: true,
-    showModal: false,
```

Comment:
These are two properties that I don't think are needed anymore. Both are set, but neither is ever read as far as I can tell.